packages/@react-spectrum/contextualhelp/docs/ContextualHelp.mdx

```diff
@@ -0,0 +1,111 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/contextualhelp';
+import {HeaderInfo, PropTable, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/contextualhelp/package.json';
+
+```jsx import
+import {ContextualHelp} from '@react-spectrum/contextualhelp';
+import {Content, Footer} from '@react-spectrum/view';
+import {Heading, Text} from '@react-spectrum/text';
+import {Link} from '@react-spectrum/link';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Overlays
+keywords: [popover]
+after_version: 3.0.0
```

Comment:
I think we want

```suggestion
after_version: 3.0.0-alpha.1
```

Reply:
Actually I think this will go straight to 3.0.0 so this should be fine.